packages/core/src/ctype/CType.ts

```diff
@@ -71,21 +56,34 @@ export function getHashForSchema(
  * @param hash CType hash as hex string.
  * @returns Schema id uri.
  */
-export function getIdForCTypeHash(
-  hash: ICType['hash']
-): ICType['schema']['$id'] {
+export function getIdForCTypeHash(hash: CTypeHash): ICType['$id'] {
```

Comment:
Since this is likely to be used as `CType.getIdForCTypeHash()` I’d prefer to rename it to `getIdForHash`